services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -179,15 +190,27 @@ public Response getReport(@Parameter(description = "result fromDate", required =
                             String fromDate,
                             @Parameter(description = "result toDate", required = true)
                             @QueryParam("toDate")
-                            String toDate) {
+                            String toDate,
+                            @Parameter(description = "Identifier of the user that will be used to filter achievements", required = false)
+                            @QueryParam("earnerId")
+                            long earnerId) {
+    if (StringUtils.isBlank(fromDate)) {
```

Comment:
Please delete `getReport` method which is a duplication of `getAllRealizations` and just add a `QueryParam` in `getAllRealizations` to indicate whether to return `xls` or `json` format.